src/components/templates/Jazzbar/components/JazzBarRoom/VideoErrorModal.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { FC } from "react";
```

Comment:
This file seems to duplicate our generic component `src/components/organisms/Room/VideoErrorModal.tsx`

What's the reasoning/need for duplicating it?

According to my diff, they are exactly the same:

```
⇒  git diff HEAD:src/components/organisms/Room/VideoErrorModal.tsx HEAD:src/components/templates/Jazzbar/components/JazzBarRoom/VideoErrorModal.tsx
```

Reply:
deleted in https://github.com/sparkletown/sparkle/pull/1135/commits/0aa3e561fb259e930158b63b5f6757b11be37d7d